### DISCUSSION TITLE
Route outgoing mails to specific (non-MX) address for certain domains

### DISCUSSION THREAD

**Discussion by VA1DER** [postfixadmin/postfixadmin#863] [Q&A]
I have a couple of VPSs each with their own virtual domains.  I want to set them up so that when they mail each other's domains that they use an internal private IP rather than the public MX IP in those domain's zones..

I should be able to use a transport map for this, but it's unclear to me how it works in PostfixAdmin.  I don't have the transport map query configured at the moment.  I see in #292 (marked as completed) that there is metion of a transport table, but there is no such table.  In #321 a transport query was added to the documentation, but it's unclear (to me) how to use it.  #350 suggests usage in some way tied to BackupMX.

Is what I'm looking for possible if I add the mysql_transport_maps.cf query from the documentation?

**Comment by SimonH-user** [comment]
It's somewhat dated and I don't know what may have changed, but I did something similar a while back. It's [detailed on the old SF wiki](https://sourceforge.net/p/postfixadmin/wiki/Relay_domains/).
Simon
